osis-core/src/main/java/com/scality/osis/utils/ScalityConstants.java

```diff
@@ -30,6 +30,10 @@ private ScalityConstants() {
 
     public static final String FILTER_KEY_VALUE_SEPARATOR = "==";
 
+    public static final String REPO_KEY_SEPARATOR = "__";
+
+    public static final String REPO_VALUE_SEPARATOR = ";;";
```

Comment:
Shouldn't this be `::`

Reply:
this was an unused variable. removed in 6a2f63423cce10658fa40363429704570b7528fb